plotly/figure_factory/_streamline.py

```diff
@@ -354,6 +354,7 @@ def get_streamline_arrows(self):
         dif_x = arrow_end_x - arrow_start_x
         dif_y = arrow_end_y - arrow_start_y
 
+        np.seterr(divide='ignore', invalid='ignore')
```

Comment:
Thanks @Kully!  I'd like to see if we can also restore the original numpy error state. Could we do something like:

```python
orig_err = np.geterr()
np.seterr(divide='ignore', invalid='ignore')
streamline_ang = np.arctan(dif_y / dif_x)
np.seterr(**orig_err)
```

Reply:
I like that idea. Let's try it.